cmd/collector/Dockerfile

```diff
@@ -5,14 +5,14 @@ FROM $base_image AS release
 ARG TARGETARCH
 ARG USER_UID=10001
 COPY collector-linux-$TARGETARCH /go/bin/collector-linux
-EXPOSE 14250/tcp
+EXPOSE 14250/tcp 14268/tcp 9411/tcp 4317/tcp 4318/tcp
```

Comment:
it would be nice to do that in the same style as all-in-one, with comments explaining each port

Reply:
Makes sense. I have pushed the changes